department-of-veterans-affairs/caseflow
# Queue | Move VACOLS issue descriptions, remand reasons into shared constant

Resolves #5194

### Description
Moves `ISSUE_INFO` and `REMAND_REASONS` into the new shared constants format

### Acceptance Criteria 
- [ ] Add/Edit Issue View and Remand Reasons screen behaves as expected

### Testing Plan
1. Go to `/queue`, begin Draft Decision checkout flow
2. Edit issue, confirm dropdown options appear as before
3. Select "Remanded" as disposition, go to Remand Reasons screen, confirm reasons appear as expected
